IBM/carbon-addons-iot-react
# feat(table): add custom cell renderers

**Summary**

- Add a custom cell renderer render prop to each column that can be optionally passed

**Change List (commits, features, bugs, etc)**

- refactor: renamed a propType to TableRowPropTypes
- feat: added renderDataFunction prop to TableBodyRow and Table to allow passing custom renderers
- test: added testcases to cover

**Related Issues**



- Closes IBM/carbon-addons-iot-react#88
- Closes IBM/carbon-addons-iot-react#50
- Closes IBM/carbon-addons-iot-react#76

**Acceptance Test (how to verify the PR)**

- Verify the status field of the StatefulTable story
- Verify the new custom cell renderer story
